components/RadioButtons.tsx

```diff
@@ -33,24 +32,25 @@ const RadioButtons = ({
 
   return (
     <>
-      {title && <h3>{title}</h3>}
       <fieldset
+        aria-labelledby={`${uuid}-legend`}
         className={styles.container}
         disabled={disabled}
-        onChange={
-          (e) =>
-            updateSelectedOption(parseInt((e.target as HTMLInputElement).value))
-          // eslint-disable-next-line react/jsx-curly-newline
-        }
+        onChange={(e) => {
+          const selectedValue = parseInt((e.target as HTMLInputElement).value)
+          updateSelectedOption(selectedValue)
+        }}
       >
+        {title && <legend id={`${uuid}-legend`}>{title}</legend>}
```

Comment:
Were there any styles associated with the now removed `h3`? Are we sure we can safely make this change?

Reply:
The styling changed e.g. not bold anymore, but I think that from now on we can some bold styling to the legend instead to act like an h3

Reply:
I couldn't find h3 styling but I think it's getting inherited by bootstrap/react/browser. I read on my a11y tool that a11y doesn't like having `h3` without `h1`/ `h2` preceding it
![image](https://user-images.githubusercontent.com/62163307/232556920-be98aac2-3952-4f6b-9e17-4d5fe8a95be5.png)
![image](https://user-images.githubusercontent.com/62163307/232556956-2a8b51d4-8d09-4fd3-83be-35131b6512bf.png)
so maybe styled legend is way to go or wrap it with h2 but that seems weird